src/lib/controllers/Client/index.js

```diff
@@ -0,0 +1,14 @@
+class ClientController {
+  static disconnectionHandler() {
```

Comment:
Why static ?

Reply:
I use static keywork because this : Expected 'this' to be used by class method 'disconnectionHandler'. (class-methods-use-this)
When i don't use this on method funtion eslint throw this error. 
I am currently in a problem loop :
use function doesn't respect paradigm > use class method doesn't respect this method rule > use static method seem not good ....
What do I have to do sensei ?

Reply:
Okay, it's just for now. This will probably move to non-static 😊